app/uk/gov/hmrc/bindingtariffclassification/service/CaseService.scala

```diff
@@ -49,7 +51,17 @@ class CaseService @Inject()(caseRepository: CaseRepository, sequenceRepository:
   }
 
   def deleteAll(): Future[Unit] = {
-    caseRepository.deleteAll
+    caseRepository.deleteAll()
+  }
+
+  def incrementDaysElapsedIfAppropriate(increment: Double, clock: Clock = Clock.systemDefaultZone()): Future[Int] = {
```

Comment:
It looks like this is going  to increase enough to have an own service/object. These logic should be isolated .

Reply:
The only extra will be a call to the Bank holiday service

Reply:
Can always pull it out as part of later work if it grows